homeassistant/components/lcn/climate.py

```diff
@@ -0,0 +1,139 @@
+"""Support for LCN climate control."""
+
+from homeassistant.components.climate import ClimateDevice, const
+from homeassistant.const import (
+    ATTR_TEMPERATURE, CONF_ADDRESS, CONF_UNIT_OF_MEASUREMENT)
+
+from . import LcnDevice, get_connection
+from .const import (
+    CONF_CONNECTIONS, CONF_LOCKABLE, CONF_MAX_TEMP, CONF_MIN_TEMP,
+    CONF_SETPOINT, CONF_SOURCE, DATA_LCN)
+
+DEPENDENCIES = ['lcn']
+
+
+async def async_setup_platform(hass, hass_config, async_add_entities,
+                               discovery_info=None):
+    """Set up the LCN climate platform."""
+    if discovery_info is None:
+        return
+
+    import pypck
```

Comment:
We can now import requirements at the top of the module, also a consequence of manifest.json.

Reply:
See also #23321